db/migrate/20200518205250_create_doorkeeper_tables.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+class CreateDoorkeeperTables < ActiveRecord::Migration[5.2]
+  def change
+    create_table :oauth_applications do |t|
+      t.string  :name,    null: false
+      t.string  :uid,     null: false
+      t.string  :secret,  null: false
+      t.integer :owner_id, null: false
+      t.string :owner_type, null: true
```

Comment:
We can probably remove this - ATM it's always "User"

Reply:
If we remove it, will that screw up the doorkeeper internals?  (I.e., does it expect this field to be present?)

Reply:
Not sure, need to test